spring-amqp/src/main/java/org/springframework/amqp/support/converter/Jackson2JsonMessageConverter.java

```diff
@@ -57,6 +57,7 @@ public Jackson2JsonMessageConverter() {
 	public Jackson2JsonMessageConverter(String... trustedPackages) {
 		this(new ObjectMapper(), trustedPackages);
 		this.objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+		this.objectMapper.findAndRegisterModules();
```

Comment:
We prefer to do this without a `ServiceLoader` and reflection.
In Spring Integration we do this: https://github.com/spring-projects/spring-integration/blob/main/spring-integration-core/src/main/java/org/springframework/integration/support/json/Jackson2JsonObjectMapper.java#L87

We do something similar in Spring for Apache Kafka: https://github.com/spring-projects/spring-kafka/blob/main/spring-kafka/src/main/java/org/springframework/kafka/support/JacksonUtils.java#L59